### DISCUSSION TITLE
Brand NEw User - Question on Response Times

### DISCUSSION THREAD

**Discussion by alansinger** [Budibase/budibase#8924] [Help]
Hi - Very new user and very excited about the promise of Budibase after watching many of the videos.

I have built a very simple app using the Budibase database - 4 tables - person, organisation , person-organisation Role, Industry

My question is in relation to the time taken from clicking on a record in the list screen and the time it tales to render a modal edit screen for that record.
It is consistently taking 5 seconds and I am wondering if this is "normal" or if there are ways I can speed up response times. I am located in Melbourne Australia

**Comment by melohagan** [comment]
Hey @alansinger 

The modals are a little bit slow as they load an entire screens worth of content, although they shouldn't be taking that long.
In the next release you'll be able to make use of side panels which are considerably faster for editing table row data!

My guess is that a lot of related data is being loaded unnecessarily. This discussion might give some further context: https://github.com/Budibase/budibase/discussions/5299
